docker/Dockerfile

```diff
@@ -44,7 +44,7 @@ RUN ln -s /usr/local/lib/python3.10/dist-packages/tensorrt_libs/libnvinfer.so.*
 ENV LD_LIBRARY_PATH=$TRT_LIBPATH:$LD_LIBRARY_PATH
 
 # Find and install requirements.txt files for all examples
-COPY . /workspace/TensorRT-Model-Optimizer
+RUN git clone https://github.com/NVIDIA/TensorRT-Model-Optimizer
```

Comment:
Thanks for the suggestions. However the reason we didn't keep it this way is because of our internal development and testing flow where we want to test internal features before releasing publicly. We may consider doing this in the future